src/components/sdk/DeviceSelection/DeviceInput.tsx

```diff
@@ -12,47 +18,58 @@ interface Props extends BaseSdkProps {
   label: string;
   notFoundMsg: string;
   devices: DeviceType[];
-  selectedDeviceId: SelectedDeviceId;
+  selectedDevice:
+    | Device
+    | AudioTransformDevice
+    | VideoTransformDevice
+    | null
+    | undefined;
   onChange: (deviceId: string) => void;
```

Comment:
So for this `onChange` prop, should we now vend out the `selectedDevice` with type not only `string`?

Reply:
NVM, I think this `onChange` can only vend selected `string` from select element.

Reply:
Yeah, can't do that because It is based on `FormField` and the value can only be `string`.